### DISCUSSION TITLE
How do you replace Prism.Interactivity in .net 5

### DISCUSSION THREAD

**Discussion by Cpt-Falcon** [PrismLibrary/Prism#2499] [WPF]
Greetings,

There's almost no information on upgrading Prism.Interactivity in .net 5. How do you go about replacing interactivity with an alternative or is there something I missed on including a .net core compatible Prism.Interactivity?

Until then it is impossible to upgrade our current app. If this is not yet supported, is there a timeline on when it will be supported? Thank you for your time.

**Comment by brianlagunas** [comment]
Interactivity is dead and is never coming back. It has been replaced with the dialog service

https://prismlibrary.com/docs/wpf/dialog-service.html
